spec/defines/admin_spec.rb

```diff
@@ -5,6 +5,11 @@
     context "on #{os}" do
       let(:facts) { os_facts }
       let(:title) { 'help' }
+      let(:pre_condition) do
```

Comment:
The alternative to this was to add the parameter to the `let(:params)` block. That was the intended way of this file and it would be good to keep that. There's a block `context 'with inheritance' do` that is testing it with inheritance.